File: OPENAPI-REST-API/openapi-client/scala-gatling/src/gatling/scala/org/openapitools/client/model/QuickserverOrderVersionCentosstream8.scala

package org.openapitools.client.model


case class QuickserverOrderVersionCentosstream8 (
    /* Version details of CentOS Stream 8. */
    _centosstream_8: Option[String]
)
object QuickserverOrderVersionCentosstream8 {
    def toStringBody(var_centosstream_8: Object) =
        s"""
        | {
        | "centosstream_8":$var_centosstream_8
        | }
        """.stripMargin
}

File: OPENAPI-REST-API/openapi-client/php-dt/src/App/DTO/QuickserverOrderVersionCentosstream8.php
<?php
declare(strict_types=1);

namespace App\DTO;

use Articus\DataTransfer\PhpAttribute as DTA;

class QuickserverOrderVersionCentosstream8
{
    /**
     * Version details of CentOS Stream 8.
     */
    #[DTA\Data(field: "centosstream-8", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $centosstream_8 = null;

}
